mayan/apps/documents/forms/review_forms.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from ..models.reviewer import Reviewer
+from ..models.applicant import Applicant
+from ..models.review import Review
+
+
+class ReviewForm(forms.Form):
+    reviewer = forms.CharField(label='Reviewer Name', max_length=30)
+    applicant = forms.CharField(label='Applicant Name', max_length=30)
```

Comment:
do we need this? Shouldn't applicant be tied to the document, while reviewer should just be the current user

Reply:
I've changed to tie the reviewer to the current user, but we have to find a way to retrieve the applicant user from the document